extensions/jitsi/common/client.mjs

```diff
@@ -26,6 +28,25 @@ class JitsiContainer {
   }
 };
 
+class JitsiOverlay extends Overlay {
```

Comment:
Maybe `JitsiFullscreenOverlay`? Possibly add jsdoc to specify that it's for video in fullscreen mode.